app/models/partner.rb

```diff
@@ -311,10 +311,43 @@ def owned_neighbourhood_ids
     neighbourhood_ids
   end
 
+  def neighbourhood_name_for_site(badge_zoom_level)
+    if service_areas.count.positive?
+      if service_areas.count > 1
+        'various'
```

Comment:
```suggestion
        'Various'
```

Reply:
theres perhaps no need for a nested if here - could just do elsifs for >1, 1, and 0

Reply:
for some reason rubocop is losing its mind over an elsif here so I'm leaving the nested statement for now, but will change the string.

Reply:
ha, when i last used rubocop it was losing its mind over too many levels of indentation, sigh